code/mobility-services/mobility-services.js

```diff
@@ -122,4 +135,4 @@ $(document).on('knack-view-render.view_41', function(event, page) {
 $(document).on('knack-view-render.view_57', function(event, page) {
   // create large CUSTOMER PORTAL button on the PORTAL page
     bigButton('available-services', 'view_57', "https://atd.knack.com/mobility-services#portal/", "arrow-right", "Mobility Services Portal");
-});
\ No newline at end of file
+});
```

Comment:
another stray closing bracket on line 138? What existing code is this closing?

Reply:
line 135